WorkAndFun/src/main/resources/templates/announcementDetails.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/html">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!--/* Bootstrap CSS */-->
+    <link th:href="@{/css/bootstrap.min.css}" rel="stylesheet"/>
+
+    <link th:href="@{/css/style.css}" rel="stylesheet"/>
+
+    <title>Szczegóły ogłoszenia</title>
+</head>
+
+
+<body class="bg-light p-3">
+<div class="flex">
+    <div class="container-lg">
+        <div class="row mb-1">
+            <div class="col-md-12" th:insert="~{fragments/navbar :: nav}">
+            </div>
+        </div>
+        <div class="row">
+            <div class="m-2 col-lg-6 bg-default">
+
+                <h1 class="m-3 text-center"><I>Szczegóły ogłoszenia:</I></h1>
+                <p class="m-2 font-weight-bold text-center" th:text="${allDetails.getType().getValue() + ': ' + allDetails.getServiceType().getServiceTypeName()}"></p>
+<!--                <p class="m-2" th:text="${allDetails.getServiceType().getServiceTypeName()}"></p>-->
+
+                <h4 class="m-2 border-info rounded bg-light text-center" th:text="${allDetails.getHeader()}"></h4>
+
+                <p class="m-2 font-weight-bold text-center"> Opis: </p>
+                <p class="m-2 border-info rounded bg-light" th:text="${allDetails.getDescription()}"></p>
+
+
+                <footer class="text-dark text-decoration-none mt-lg-4">
+                    <hr>
+                    <div class="row align-bottom">
+                        <div class="col-md-3">
+                            <ul class="list-unstyled">
+                                <li>
+                                    <p class="m-1" style="font-size: x-small">id ogłoszenia:</p>
```

Comment:
w html nie piszę się `style="..."` tylko jest to dodawane do CSS a w html korzysta się z dodanej klasy CSSowej

Reply:
done